apps/nextjs-website/src/components/rendering/MDtoJSX.tsx

```diff
@@ -0,0 +1,13 @@
+import { marked } from 'marked';
+import parse from 'html-react-parser';
+import { Typography } from '@mui/material';
+import { Variant } from '@mui/material/styles/createTypography';
+
+export const MDtoJSX = (
```

Comment:
I would change the name and move into `src/components`

```suggestion
export const MarkdownRenderer = (
```